docs/jwst/flatfield/reference_files.rst

```diff
@@ -78,45 +101,49 @@ there are different flat fields for fixed-slit data, IFU data, and for
 multi-object spectroscopic data.  Here is a summary of the contents of these
 files.
 
-For the fore optics, the flat field for fixed-slit data contains just a
+For the fore optics (FFLAT), the flat field for fixed-slit data contains just a
 FAST_VARIATION table (i.e. there is no image).  This table has five rows,
-one for each of the fixed slits.  The flat field for IFU data also contains
+one for each of the fixed slits.  The FFLAT for IFU data also contains
 just a FAST_VARIATION table, but it has only one row with the value "ANY"
-in the "slit_name" column.  For multi-object spectroscopic data, the flat
-field contains four sets (one for each MSA quadrant) of images, WAVELENGTH
-tables, and FAST_VARIATION tables.  The images are unique to the fore
-optics flat fields, however.  The image "pixels" correspond to micro-shutter
-array slits, rather than to detector pixels.  The array size is 365 rows
+in the "slit_name" column.  For multi-object spectroscopic data, the FFLAT
+contains four sets of images (one for each MSA quadrant), WAVELENGTH
+tables, and FAST_VARIATION tables.  The images are unique to the FFLATs,
+however.  The image "pixels" correspond to micro-shutter array slits, rather
+than to detector pixels.  The array size is 365 rows
 by 171 columns, and there are multiple planes to handle the slow variation
 of flat field with wavelength.
 
-For the spectrograph optics, the flat-field files have nearly the same
+For the spectrograph optics (SFLAT), the flat-field files have nearly the same
 format for fixed-slit data, IFU, and multi-object data.  The difference is
 that for fixed-slit and IFU data, the image is just a single plane,
 i.e. the only variation with wavelength is in the FAST_VARIATION table,
 while there are multiple planes in the image for multi-object spectroscopic
 data (and therefore there is also a corresponding WAVELENGTH table, with
 one row for each plane of the image).
 
-For the detector section, the flat field file contains a 3-D image
+For the detector section, the DFLAT file contains a 3-D image
 (i.e. the flat field at multiple wavelengths), a corresponding
 WAVELENGTH table, and a FAST_VARIATION table with one row.
 
 As just described, there are 3 types of reference files for NIRSpec (FFLAT, 
 SFLAT, and DFLAT), and within each of these types, there are several formats, 
 which are now described.
 
+FFLAT Reference File
+--------------------
+
+:REFTYPE: FFLAT
 
-Fore Optics (FFLAT)
-:::::::::::::::::::
-There are 3 types of FFLAT reference files: fixed slit, msa spec, and IFU. For each type
+There are 3 forms of FFLAT reference files: fixed slit, MSA spec, and IFU. For each type
```

Comment:
Probably don't need to say it. It's a standard statement that we include for all FITS ref files. I do think the term "primary" is appropriate though. In FITS lingo "primary" usually refers to the first, regular (non-extension) HDU, i.e. the one that's formatted according to the original FITS specs. The term "primary" was adopted for it when extensions were invented. I don't think it refers to the first HDU that actually has data in it. Regarding the use of "empty", I guess you're right that it's really more than just empty - it's not there at all. The only part of the primary HDU that exists is the header. I guess we could rephrase it to something like "the primary HDU does not contain a data array" (so it's just an "HU" instead of an "HDU"!).